app/stac_api/views.py

```diff
@@ -110,11 +113,14 @@ def list(self, request, *args, **kwargs):
         return Response(data)
 
 
-class CollectionDetail(generics.RetrieveAPIView):
+class CollectionDetail(generics.RetrieveAPIView, mixins.UpdateModelMixin):
```

Comment:
To be consistent I would also use `GenericAPIView` and explicitely add the `get()` method, this way we are consistent in the views. It adds a little bit more of code but it isn't too much and it make the code easier to understand in IMHO.